config-solaris.md

```diff
@@ -84,20 +84,20 @@ For additional information on properties, check the [zonecfg(1M)][zonecfg.1m_2]
 * **`lowerLink`** *(string, OPTIONAL)* Specify the link over which the VNIC will be created.
 Mapped to `lower-link` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`allowedAddress`** *(string, OPTIONAL)* The set of IP addresses that the container can use might be constrained by specifying the allowedAddress property.
-If allowedAddress has not been specified, then they can use any IP address on the associated physical interface for the network resource.
-Otherwise, when allowedAddress is specified, the container cannot use IP addresses that are not in the allowedAddress list for the physical address.
-Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
+    If `allowedAddress` has not been specified, then they can use any IP address on the associated physical interface for the network resource.
+    Otherwise, when allowedAddress is specified, the container cannot use IP addresses that are not in the allowedAddress list for the physical address.
+    Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`configureAllowedAddress`** *(string, OPTIONAL)* If configureAllowedAddress is set to true, the addresses specified by allowedAddress are automatically configured on the interface each time the container starts.
```

Comment:
If you're backticking `allowdAddress` (like you do a few lines above), you probably want to get it (and `configureAllowedAddress`?) here too.  Or we can punt some/all of the backtick fixup to a follow-up PR.